plasmapy/formulary/parameters.py

```diff
@@ -1356,11 +1456,17 @@ def inertial_length(n: u.m ** -3, particle: particles.Particle) -> u.m:
     return c / omega_p
 
 
+lp_ = inertial_length
+""" Alias to :func:`inertial_Length`. """
```

Comment:
@namurphy @StanczakDominik Are we good with this alias?  I'm thinking `d_inertial_` and/or `cwp_` would be more appropriate.

Reply:
No strong preference here!

Reply:
I'm also polling the group here at UCLA.  Not much feedback yet, but it's looking like `cwp_` is the best candidate.

Reply:
I'll go for cwp_ then. 7421364 should resolve this thread.